src/main/java/gov/usds/case_issues/config/WebConfig.java

```diff
@@ -95,6 +106,27 @@ public CookieSerializer getCookieSerializer() {
 		return serializer;
 	}
 
+	/**
+	 * If desired, configure the embedded Tomcat server to listen on an additional HTTP port.
+	 * (To enable both HTTPS and HTTP connectors, use Spring Boot's built-in configuration for
+	 * the HTTPS port, so as to take advantage of auto-configuration.)
+	 */
+	@Bean
+	@ConditionalOnProperty("web-customization.additional-http-port")
+	public WebServerFactoryCustomizer<TomcatServletWebServerFactory> getServerCustomizer() {
+		int port = _customProperties.getAdditionalHttpPort();
+		if (port <= 0) {
+			throw new IllegalArgumentException(
+				String.format("Configured additional HTTP port (%d) is not valid", port)
```

Comment:
Interesting question: what happens if this is zero? Options are “tomcat throws an exception” and “we get a random available port,” which are very different outcomes. Not clear that doing the range check ourselves is meaningful other than out of a general spirit of good citizenship.